modules/round/src/main/JsonView.scala

```diff
@@ -167,7 +168,8 @@ final class JsonView(
             "player" -> {
               commonWatcherJson(game, player, playerUser, withFlags) ++ Json.obj(
                 "version"   -> socket.version.value,
-                "spectator" -> true
+                "spectator" -> true,
+                "id" -> owner.option(pov.playerId)
```

Comment:
I don't think you need to add the `owner` argument. `me.flatMap(game.player).map(_.id)` should do.

Reply:
Ah, much easier. Thanks!